### PR TITLE
Sprint1308 3008/change svg imports

### PR DESCRIPTION
**Pull request by aydin101**

### CODE REVIEW

File: ui_src/src/components/sideBar/style.scss [hunk 1]
```diff
@@ -86,7 +86,8 @@
                 margin: 0;
                 cursor: pointer;
             }
-            img {
+            img,
```
Comment (avrhamNeeman) [inline ui_src/src/components/sideBar/style.scss:89]: The same here, img tag is unnecessary.

File: ui_src/src/domain/administration/billing/requests/style.scss [hunk 2]
```diff
@@ -75,7 +75,8 @@
                             font-family: 'InterMedium';
                             font-size: 16px;
                             align-self: center;
-                            img {
+                            img,
```
Comment (avrhamNeeman) [inline ui_src/src/domain/administration/billing/requests/style.scss:78]: The same

File: ui_src/src/domain/administration/integrations/components/dataDogIntegration/style.scss [hunk 3]
```diff
@@ -1,66 +1,67 @@
...
-            a{
+            a {
                 color: var(--purple);
             }
-            a:hover{
+            a:hover {
                 text-decoration: underline;
             }
-            .img{
+            .img,
```
Comment (avrhamNeeman) [inline ui_src/src/domain/administration/integrations/components/dataDogIntegration/style.scss:57]: the same

File: ui_src/src/domain/administration/integrations/components/integrationItem/style.scss [hunk 4]
```diff
@@ -1,54 +1,54 @@
...
+        position: absolute;
         margin: 10px;
         box-shadow: 0px 4px 8px rgba(38, 38, 38, 0.2);
         border-radius: 8px;
         background: var(--purple);
         padding: 3px 7px;
         color: white;
-        img{
-           width: 18px; 
+        img,
```
Comment (avrhamNeeman) [inline ui_src/src/domain/administration/integrations/components/integrationItem/style.scss:27]: .

File: ui_src/src/domain/administration/integrations/style.scss [hunk 5]
```diff
@@ -81,7 +81,8 @@
             display: flex;
             align-items: center;
             justify-content: center;
-            img {
+            img,
```
Comment (avrhamNeeman) [inline ui_src/src/domain/administration/integrations/style.scss:84]: .

File: ui_src/src/domain/administration/style.scss [hunk 6]
```diff
@@ -29,7 +29,8 @@
                 font-family: 'InterSemiBold';
                 font-size: 14px;
                 cursor: pointer;
-                img {
+                img,
+                svg {
```
Comment (avrhamNeeman) [inline ui_src/src/domain/administration/style.scss:33]: .

File: ui_src/src/domain/functions/components/functionList/style.scss [hunk 7]
```diff
@@ -49,7 +49,8 @@
             height: 100%;
             justify-content: center;
             width: 100%;
-            img {
+            img,
```
Comment (avrhamNeeman) [inline ui_src/src/domain/functions/components/functionList/style.scss:52]: .

File: ui_src/src/domain/login/style.scss [hunk 8]
```diff
@@ -18,7 +18,8 @@
             grid-template-rows: 25% 25% 50%;
             .logoImg {
                 padding: 50px;
-                img {
+                img,
```
Comment (avrhamNeeman) [inline ui_src/src/domain/login/style.scss:21]: .

File: ui_src/src/domain/messageJourney/style.scss [hunk 9]
```diff
@@ -4,81 +4,81 @@
...
         justify-content: space-between;
         width: 100%;
-        p{
+        p {
             margin: 0;
             font-family: 'InterSemiBold';
             font-size: 14px;
         }
-        img{
+        img,
```
Comment (avrhamNeeman) [inline ui_src/src/domain/messageJourney/style.scss:45]: .

File: ui_src/src/domain/overview/genericDetails/style.scss [hunk 10]
```diff
@@ -127,100 +125,101 @@
...
                     padding: 5px;
                     height: 35px;
                     justify-content: space-between;
                     align-items: center;
                     cursor: pointer;
-                    .left{
+                    .left {
                         display: flex;
-                        img{
+                        img,
```
Comment (avrhamNeeman) [inline ui_src/src/domain/overview/genericDetails/style.scss:164]: .

File: ui_src/src/domain/overview/getStarted/createAppUser/style.scss [hunk 11]
```diff
@@ -82,7 +82,8 @@
                     font-family: 'InterSemiBold';
                 }
 
-                img {
+                img,
```
Comment (avrhamNeeman) [inline ui_src/src/domain/overview/getStarted/createAppUser/style.scss:85]: .

File: ui_src/src/domain/overview/getStarted/sideStep/index.js [hunk 12]
```diff
@@ -12,18 +12,20 @@
...
+import { ReactComponent as DataProducedIcon } from '../../../../assets/images/dataProduced.svg';
+import { ReactComponent as ConsumeDataIcon } from '../../../../assets/images/stationsIconActive.svg';
+import { ReactComponent as FullStationIcon } from '../../../../assets/images/fullStation.svg';
+import { ReactComponent as FinishFlagIcon } from '../../../../assets/images/finishFlag.svg';
+import { ReactComponent as GrayAppUserIcon } from '../../../../assets/images/grayAppUserIcon.svg';
+import { ReactComponent as GrayProduceDataIcon } from '../../../../assets/images/grayProduceDataImg.svg';
+import { ReactComponent as GrayConsumeDataIcon } from '../../../../assets/images/grayConsumeDataImg.svg';
+import { ReactComponent as GrayfinishStepIcon } from '../../../../assets/images/grayFinish.svg';
+import { ReactComponent as CompletedStepIcon } from '../../../../assets/images/checkIcon.svg';
+import Finish from '../finish';
```
Comment (avrhamNeeman) [inline ui_src/src/domain/overview/getStarted/sideStep/index.js:28]: Unneccerry import

File: ui_src/src/domain/overview/stations/style.scss [hunk 13]
```diff
@@ -169,7 +170,8 @@
         flex-direction: column;
         align-items: center;
         justify-content: center;
-        img {
+        img,
```
Comment (avrhamNeeman) [inline ui_src/src/domain/overview/stations/style.scss:173]: .

File: ui_src/src/domain/overview/style.scss [hunk 14]
```diff
@@ -103,7 +103,8 @@
                     p {
                         margin: 0;
                     }
-                    img {
+                    img,
```
Comment (avrhamNeeman) [inline ui_src/src/domain/overview/style.scss:106]: .

File: ui_src/src/domain/overview/systemComponents/style.scss [hunk 15]
```diff
@@ -61,7 +61,8 @@
                             border-radius: 4px;
                             justify-content: center;
                             align-items: center;
-                            img {
+                            img,
```
Comment (avrhamNeeman) [inline ui_src/src/domain/overview/systemComponents/style.scss:64]: .

File: ui_src/src/domain/schema/components/createSchema/style.scss [hunk 16]
```diff
@@ -10,110 +10,111 @@
...
+                .flex-title {
                     display: flex;
-                    p{
+                    p {
                         font-family: 'InterSemiBold';
                         font-size: 24px;
                         margin: 0;
                     }
-                    img{
+                    img,
```
Comment (avrhamNeeman) [inline ui_src/src/domain/schema/components/createSchema/style.scss:41]: .

File: ui_src/src/domain/schema/components/schemaBox/style.scss [hunk 17]
```diff
@@ -1,55 +1,56 @@
...
-            .is-used{
+            .is-used {
                 display: flex;
                 font-family: 'InterSemiBold';
                 font-size: 12px;
                 width: 100px;
-                place-content:end;
-                img {
+                place-content: end;
+                img,
```
Comment (avrhamNeeman) [inline ui_src/src/domain/schema/components/schemaBox/style.scss:40]: .

File: ui_src/src/domain/schema/components/schemaDetails/index.js [hunk 18]
```diff
@@ -12,22 +12,22 @@
 
 import './style.scss';
 
-import { AddRounded, CheckCircleOutlineRounded, ErrorOutlineRounded } from '@material-ui/icons';
+import { AddRounded, CheckCircleOutlineRounded, Create, ErrorOutlineRounded } from '@material-ui/icons';
```
Comment (avrhamNeeman) [inline ui_src/src/domain/schema/components/schemaDetails/index.js:15]: Unneccerry import

File: ui_src/src/domain/schema/components/schemaDetails/style.scss [hunk 19]
```diff
@@ -1,28 +1,29 @@
...
                 display: flex;
                 align-items: center;
-                .schema-json-name{
+                .schema-json-name {
                     color: #2e2c34;
                     font-size: 14px;
                 }
-                img{
-                    padding-right:6px;
+                img,
```
Comment (avrhamNeeman) [inline ui_src/src/domain/schema/components/schemaDetails/style.scss:21]: .

File: ui_src/src/domain/schema/components/schemaList/index.js [hunk 20]
```diff
@@ -29,6 +29,7 @@ import { filterArray } from '../../../../services/valueConvertor';
 import DeleteItemsModal from '../../../../components/deleteItemsModal';
 import { Link, useHistory } from 'react-router-dom';
 import pathDomains from '../../../../router';
+import { Place } from '@material-ui/icons';
```
Comment (avrhamNeeman) [inline ui_src/src/domain/schema/components/schemaList/index.js:32]: unnecessary import

File: ui_src/src/domain/schema/components/schemaList/style.scss [hunk 21]
```diff
@@ -31,7 +31,8 @@
             height: 100%;
             justify-content: center;
             width: 100%;
-            img {
+            img,
```
Comment (avrhamNeeman) [inline ui_src/src/domain/schema/components/schemaList/style.scss:34]: .

File: ui_src/src/domain/stationOverview/components/generateTokenModal/style.scss [hunk 22]
```diff
@@ -34,7 +34,8 @@
             color: var(--purple);
             width: fit-content;
             cursor: pointer;
-            img {
+            img,
```
Comment (avrhamNeeman) [inline ui_src/src/domain/stationOverview/components/generateTokenModal/style.scss:37]: .

File: ui_src/src/domain/stationOverview/components/updateSchemaModal/style.scss [hunk 23]
```diff
@@ -1,159 +1,158 @@
-.update-schema-modal-container{
```
Comment (avrhamNeeman) [inline ui_src/src/domain/stationOverview/components/updateSchemaModal/style.scss:1]: There are many modifications in this file. Is this on purpose?

File: ui_src/src/domain/stationOverview/components/useSchemaModal/index.js [hunk 24]
```diff
@@ -13,13 +13,16 @@
 import './style.scss';
 
 import React, { useContext, useEffect, useState } from 'react';
-import { AddRounded } from '@material-ui/icons';
+import { AddRounded, Place } from '@material-ui/icons';
```
Comment (avrhamNeeman) [inline ui_src/src/domain/stationOverview/components/useSchemaModal/index.js:16]: Unneccerry import

File: ui_src/src/domain/stationOverview/components/useSchemaModal/schemaItem/style.scss [hunk 25]
```diff
@@ -1,47 +1,47 @@
...
-    .name-wrapper{
+    .name-wrapper {
         display: flex;
-        .name{
+        .name {
             font-family: 'InterSemiBold';
             font-size: 14px;
         }
-        img{
+        img,
```
Comment (avrhamNeeman) [inline ui_src/src/domain/stationOverview/components/useSchemaModal/schemaItem/style.scss:17]: .

File: ui_src/src/domain/stationOverview/stationObservabilty/ProduceConsumList/style.scss [hunk 26]
```diff
@@ -28,7 +28,8 @@
         .producer-placeholder {
             display: flex;
             align-items: center;
-            img {
+            img,
```
Comment (avrhamNeeman) [inline ui_src/src/domain/stationOverview/stationObservabilty/ProduceConsumList/style.scss:31]: .

File: ui_src/src/domain/stationOverview/stationObservabilty/components/customCollapse/index.js [hunk 27]
```diff
@@ -12,10 +12,10 @@
 import './style.scss';
 
 import React, { useEffect, useState } from 'react';
-import { Collapse } from 'antd';
+import { Col, Collapse } from 'antd';
```
Comment (avrhamNeeman) [inline ui_src/src/domain/stationOverview/stationObservabilty/components/customCollapse/index.js:15]: Unneccerry import

File: ui_src/src/domain/stationOverview/stationObservabilty/components/customCollapse/style.scss [hunk 28]
```diff
@@ -69,7 +69,8 @@
         max-height: 190px;
         overflow: scroll;
         padding-right: 10px;
-        img {
+        img,
```
Comment (avrhamNeeman) [inline ui_src/src/domain/stationOverview/stationObservabilty/components/customCollapse/style.scss:72]: .

File: ui_src/src/domain/stationOverview/stationObservabilty/components/messageDetails/index.js [hunk 29]
```diff
@@ -17,16 +17,17 @@ import Lottie from 'lottie-react';
...
 import { ApiEndpoints } from '../../../../../const/apiEndpoints';
-import Journey from '../../../../../assets/images/journey.svg';
+import { ReactComponent as JourneyIcon } from '../../../../../assets/images/journey.svg';
 import { httpRequest } from '../../../../../services/http';
 import Button from '../../../../../components/button';
 import { StationStoreContext } from '../../..';
 import CustomCollapse from '../customCollapse';
 import MultiCollapse from '../multiCollapse';
 import StatusIndication from '../../../../../components/indication';
+import { Place } from '@material-ui/icons';
```
Comment (avrhamNeeman) [inline ui_src/src/domain/stationOverview/stationObservabilty/components/messageDetails/index.js:30]: Unneccerry import

File: ui_src/src/domain/stationOverview/stationObservabilty/components/multiCollapse/style.scss [hunk 30]
```diff
@@ -12,33 +12,36 @@
...
-            status{
+    .collapse-child {
+        .collapse-header {
+            p {
+                font-size: 12px !important;
+            }
+            status {
                 margin-right: 10px;
-                img {
+                img,
```
Comment (avrhamNeeman) [inline ui_src/src/domain/stationOverview/stationObservabilty/components/multiCollapse/style.scss:25]: .

File: ui_src/src/domain/stationOverview/stationObservabilty/messages/index.js [hunk 31]
```diff
@@ -356,6 +357,7 @@ const Messages = () => {
             {tabValue === tabs[0] && (stationState?.stationSocketData?.messages === null || stationState?.stationSocketData?.messages?.length === 0) && (
                 <div className="waiting-placeholder msg-plc">
                     <img width={100} src={waitingMessages} alt="waitingMessages" />
```
Comment (avrhamNeeman) [inline ui_src/src/domain/stationOverview/stationObservabilty/messages/index.js:359]: I think you missed to delete this row + the relevant import

File: ui_src/src/domain/stationOverview/stationObservabilty/messages/style.scss [hunk 32]
```diff
@@ -216,7 +216,8 @@
                 margin-left: 3%;
                 .botton-title {
                     display: flex;
-                    img {
+                    img,
```
Comment (avrhamNeeman) [inline ui_src/src/domain/stationOverview/stationObservabilty/messages/style.scss:219]: .

File: ui_src/src/domain/stationOverview/stationOverviewHeader/style.scss [hunk 33]
```diff
@@ -13,7 +13,8 @@
                 margin-right: 5px;
                 display: flex;
                 gap: 5px;
-                img {
+                img,
```
Comment (avrhamNeeman) [inline ui_src/src/domain/stationOverview/stationOverviewHeader/style.scss:16]: .

File: ui_src/src/domain/stationsList/stationBoxOverview/style.scss [hunk 34]
```diff
@@ -138,9 +138,11 @@
                     width: 30px;
                     height: 30px;
                     justify-content: center;
+                    align-items: center;
                     background: rgba(101, 87, 255, 0.2);
                     cursor: pointer;
-                    img {
+                    img,
```
Comment (avrhamNeeman) [inline ui_src/src/domain/stationsList/stationBoxOverview/style.scss:144]: .

File: ui_src/src/domain/streamLineage/components/station/style.scss [hunk 35]
```diff
@@ -1,37 +1,38 @@
...
         display: flex;
         flex-direction: column;
         align-items: center;
         gap: 10px;
         height: 100%;
-        justify-content:space-between;
+        justify-content: space-between;
         padding: 10px 0;
-        img{
+        img,
```
Comment (avrhamNeeman) [inline ui_src/src/domain/streamLineage/components/station/style.scss:28]: .

File: ui_src/src/domain/streamLineage/style.scss [hunk 36]
```diff
@@ -26,123 +26,126 @@ $grid-color: rgba(150, 150, 150, .3);
                 color: #b4b4b4;
                 font-family: 'InterMedium';
             }
-            .flex{
+            .flex {
                 display: flex;
                 gap: 5px;
-                img{
+                img,
```
Comment (avrhamNeeman) [inline ui_src/src/domain/streamLineage/style.scss:32]: .

File: ui_src/src/domain/sysLogs/components/logPayload/style.scss [hunk 37]
```diff
@@ -10,47 +10,48 @@
...
             height: 25px;
             display: flex;
-            border: 1px solid #EEEEEE;
+            border: 1px solid #eeeeee;
             border-radius: 4px;
-            align-items:center;
+            align-items: center;
             padding-right: 8px;
-            img{
+            img,
```
Comment (avrhamNeeman) [inline ui_src/src/domain/sysLogs/components/logPayload/style.scss:39]: .

File: ui_src/src/domain/users/index.js [hunk 38]
```diff
@@ -18,10 +18,15 @@ import { AccountCircleRounded } from '@material-ui/icons';
 import { LOCAL_STORAGE_USER_PASS_BASED_AUTH } from '../../const/localStorageConsts';
 import { isCloud, parsingDate } from '../../services/valueConvertor';
 import addUserIcon from '../../assets/images/addUserIcon.svg';
```
Comment (avrhamNeeman) [inline ui_src/src/domain/users/index.js:20]: Please delete all unneccerry imports

File: ui_src/src/domain/users/style.scss [hunk 39]
```diff
@@ -45,41 +45,41 @@
...
+            .user-action {
                 display: flex;
                 gap: 10px;
-                .action-button{
+                .action-button {
                     display: flex;
                     position: relative;
                     align-items: center;
-                    img{
+                    img,
```
Comment (avrhamNeeman) [inline ui_src/src/domain/users/style.scss:81]: .

**Comment by avrhamNeeman** [comment]
There seems to be an issue with the integration page as it's not loading for some reason

**Comment by avrhamNeeman** [comment]
The icon in the Schema box is not aligned with the text.

Screenshot 2023-09-13 at 22 35 07

**Comment by avrhamNeeman** [comment]
Go to schema details -> enforce to station button
Screenshot 2023-09-13 at 22 33 46

**Comment by avrhamNeeman** [comment]
Schema details icons
Screenshot 2023-09-13 at 22 32 06

**Comment by avrhamNeeman** [comment]
Unneccery img
Screenshot 2023-09-13 at 22 31 35

**Comment by avrhamNeeman** [comment]
The icons on the side bar are too small.
Screenshot 2023-09-13 at 22 29 36

**Comment by avrhamNeeman** [comment]
There are unnecessary images in the Select field on the throughput component
Screenshot 2023-09-13 at 22 29 19

File: ui_src/src/domain/administration/integrations/components/dataDogIntegration/style.scss (hunk 3 above)
Comment (aydin101) [reply ui_src/src/domain/administration/integrations/components/dataDogIntegration/style.scss:57]: It is needed in the index file. it is used in png file

File: ui_src/src/domain/schema/components/schemaDetails/index.js (hunk 18 above)
Comment (aydin101) [reply ui_src/src/domain/schema/components/schemaDetails/index.js:15]: btw I have not added this part but I will remove it

File: ui_src/src/components/detailBox/index.js [hunk 40]
```diff
@@ -12,12 +12,13 @@
...
 import React from 'react';
 
-const DetailBox = ({ img, title, desc, data, children, rightSection = true }) => {
+const DetailBox = ({ icon, title, desc, data, children, rightSection = true }) => {
     return (
         <div className="detail-box-container">
             <div className="detail-box-wrapper">
                 <div className="detail-img">
-                    <img width={24} src={img} alt="leader" />
+                    {/* <img width={24} src={img} alt="leader" /> */}
```
Comment (avrhamNeeman) [inline ui_src/src/components/detailBox/index.js:20]: remove comment

File: ui_src/src/domain/stationOverview/stationObservabilty/messages/index.js [hunk 41]
```diff
@@ -372,15 +376,17 @@ const Messages = () => {
             {tabValue === tabs[2] && (
                 <div className="details">
                     <DetailBox
-                        img={dlsEnableIcon}
+                        // img={dlsEnableIcon}
```
Comment (avrhamNeeman) [inline ui_src/src/domain/stationOverview/stationObservabilty/messages/index.js:379]: remove all comments

**Review by avrhamNeeman — APPROVED** [review]

**Review by idanasulinmemphis — APPROVED** [review]

**Pull request merged**